src/AzureClient/AzureExecutionTarget.cs

```diff
@@ -64,7 +68,7 @@ protected AzureExecutionTarget(string? targetId)
         ///     Valid target IDs are structured as "provider.target".
         ///     For example, "ionq.simulator" or "honeywell.qpu".
         /// </remarks>
-        protected static AzureProvider? GetProvider(string? targetId)
+        protected internal static AzureProvider? GetProvider(string? targetId)
```

Comment:
Having this method as `protected internal` instead of `protected` is what allows [this line](https://github.com/microsoft/iqsharp/blob/78ecc32e904db972078416b0589e31cddd4467d4/src/AzureClient/AzureClient.cs#L47) to reference the function.